dist/systemd/obsapisetup.service

```diff
@@ -0,0 +1,11 @@
+Description=OBS API Setup
+After=network.target
+         
+[Service]
+EnvironmentFile=/etc/sysconfig/obs-server
+Type=oneshot
+ExecStart=/usr/lib/obs/server/setup-appliance.sh --non-interactive --setup-only
+ExecReload=/bin/kill -HUP $MAINPID
```

Comment:
What is the intention behind an reload of setup-appliance.sh? I´m quite sure that we do not trap SIGHUP.